modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -272,35 +223,49 @@ Remember that since this batch aggregator is streaming, you can only access its
 </batch:job>
 ----
 
-Due to memory restrictions, random access is not supported for streaming aggregators. +
-The record payloads for random access are exposed as an `immutable List`, and since streaming aggregators implies having access to the entire set of records, without a fixed commit size, the runtime can't guarantee that all records will fit in memory.
+To learn more about processing within the Batch Aggregation scope and how aggregation differs from processing within the Batch Step scope, see xref:batch-processing-concept.adoc#phase_process[Process Phase].
 
-==== Tips
+Streaming Tips:
 
-* *Streaming from SaaS providers:* In general, you likely wouldn't use batch streaming when sending data through an Anypoint Connector to a SaaS provider like Salesforce, because SaaS providers often have restrictions on accepting streaming input. Use streaming batch processing when writing to a file such as CSV, JSON, or XML.
+* Streaming from SaaS providers: In general, you are unlikely to use batch streaming when sending data through an Anypoint Connector to a SaaS provider, such as Salesforce, because SaaS providers often have restrictions on accepting streaming input.
 
-* *Batch streaming and performance:* Batch processing streaming data does affect the performance of your application, slowing the pace at which it processes transactions. Though performance slows, the trade-off to be able to batch process streaming data may warrant using it in your implementation.
+* Batch streaming is often useful in batch processing when writing to a file such as CSV, JSON, or XML.
 
-* *Batch streaming and access to items:* The biggest drawback to using batch streaming is that you have limited access to the items in the output. In other words, with a _fixed-size commit_, you get an unmodifiable list, thus allowing you to access and iteratively process its items; with _streaming commit_, you get a one-read, forward-only iterator.
+* Batch streaming and performance: Batch streaming affects the performance of your application because the array of records is stored in RAM, which can slow the pace at which transactions process. Though performance slows, the trade-off of streaming data can warrant using it in your implementation.
 
-=== Batch Aggregator Characteristics
+* Batch streaming and access to items: The biggest drawback to using batch streaming is the limitation on access to the items in the output. A fixed-size commit provides an unmodifiable list of records,
+//TODO: IS "an unmodifiable" correct here or is it "a modifiable"
```

Comment:
//TODO: IS "an unmodifiable" correct here or is it "a modifiable"
//TODO: What do we mean by "commit" here? as in "fixed-size commit" and "streaming
//commit". Should we use that term? Could we use a different term if we wanted? If ////so, what term?